core/src/main/scala/org/apache/spark/shuffle/hash/HashShuffleReader.scala

```diff
@@ -33,11 +33,34 @@ private[spark] class HashShuffleReader[K, C](
     "Hash shuffle currently only supports fetching one partition")
 
   private val dep = handle.dependency
+  private val blockManager = SparkEnv.get.blockManager
 
   /** Read the combined key-values for this reduce task */
   override def read(): Iterator[Product2[K, C]] = {
+    val blockStreams = BlockStoreShuffleFetcher.fetchBlockStreams(
+                         handle.shuffleId, startPartition, context)
```

Comment:
sorry to nit on this again, but I actually meant indented two spaces more than the `val`, not two spaces more than the `=`

```
val blockStreams = BlockStoreShuffleFetcher.fetchBlockStreams(
  handle.shuffleId, startPartition, context)
```